### ISSUE TITLE
storage Adapter Error

### ISSUE THREAD

**Issue by sudhirkumar1980** [mspnp/iot-guidance#10]
I keep getting this error after deploying storage adapter on AKS.

kubectl -n dronemgmt logs -l app=storageadapter
[WebService.e2b572b9-1be3-4716-befb-9da27e6b2d79][2020-05-13 17:19:21Z][ERROR][ConfigData:ProcessMandatoryPlaceholders] Environment variables not found, {"varsNotFound":",PCS_AAD_APPID"}

Unhandled Exception: Microsoft.Azure.IoTSolutions.StorageAdapter.Services.Exceptions.InvalidConfigurationException: Environment variables not found: , PCS_AAD_APPID
   at Microsoft.Azure.IoTSolutions.StorageAdapter.Services.Runtime.ConfigData.ProcessMandatoryPlaceholders(String& value) in /home/azureiotpcs/vsts-agent/_work/20/s/storage-adapter/Services/Runtime/ConfigData.cs:line 158
   at Microsoft.Azure.IoTSolutions.StorageAdapter.Services.Runtime.ConfigData.ReplaceEnvironmentVariables(String& value, String defaultValue) in /home/azureiotpcs/vsts-agent/_work/20/s/storage-adapter/Services/Runtime/ConfigData.cs:line 125
   at Microsoft.Azure.IoTSolutions.StorageAdapter.Services.Runtime.ConfigData.SetUpKeyVault() in /home/azureiotpcs/vsts-agent/_work/20/s/storage-adapter/Services/Runtime/ConfigData.cs:line 82
   at Microsoft.Azure.IoTSolutions.StorageAdapter.WebService.Program.Main(String[] args) in /home/azureiotpcs/vsts-agent/_work/20/s/storage-adapter/WebService/Program.cs:line 16
/app/run.sh: line 7:     6 Aborted                 (core dumped) dotnet Microsoft.Azure.IoTSolutions.StorageAdapter.WebService.dll

**Comment by magrande** [comment]
@sudhirkumar1980   Thank you for raising this issue. We were able to repro it and we figured out that as of December 10th 2020, the remote monitoring and predictive maintenance accelerators have been removed from the Azure IoT solution accelerators site and are no longer available for new deployments. The GitHub repositories for both the accelerators have been archived. The code is still available for anyone to access but the repositories aren't taking any new contributions.

[IoT solution accelerators FAQ - Azure | Microsoft Docs](https://docs.microsoft.com/en-us/azure/iot-accelerators/iot-accelerators-faq)
